### ISSUE TITLE
Launching t-rex-64 script causes error

### ISSUE THREAD

**Issue by jgolbez** [cisco-system-traffic-generator/trex-core#591]
After running dpdk_setup_ports.py, verifying cfg yaml file and traffic yaml file exist, when launching t-rex-64 script get this output:

$ sudo ./t-rex-64 --cfg /etc/br1_to_dc1.yaml -f cap2/imix-br1_to_dc1.yaml -d 30 -m 1 -c 1 -k 5 --nc
The ports are bound/configured.
./t-rex-64: line 88: ./_t-rex-64: No such file or directory

It appears to be inserting an underscore as part of the launch which breaks the location of the script while running

![image](https://user-images.githubusercontent.com/42747999/101060573-8daba880-355d-11eb-8d1a-c4241c4290cc.png)
![image](https://user-images.githubusercontent.com/42747999/101060609-98fed400-355d-11eb-9b5f-57229eb38e22.png)

**Comment by jgolbez** [comment]
After experimenting it appears to be in this part of the bash script at the bottom where this is happening:

![image](https://user-images.githubusercontent.com/42747999/101063101-6e624a80-3560-11eb-9277-d9f756856008.png)

However if you fix the _ what happens instead s we get into an echo loop where it says the ports are bound.

![image](https://user-images.githubusercontent.com/42747999/101063189-889c2880-3560-11eb-8234-160e5019997d.png)

**Comment by hhaim** [comment]
@jgolbez try to read the wiki, especially the part how to build it

**Comment by jgolbez** [comment]
I am using Ubuntu Desktop on ESXI here but the build configure script references a package that's not clear, and there is no requirements.txt

This was the error I received when doing the build configure, I cant find any documentation on required packages for t-rex and this used to work fine unsure when the version change caused this to happen or if Ubuntu removed a base package


![image](https://user-images.githubusercontent.com/42747999/101065926-c0f13600-3563-11eb-8ce5-890e8d541c5e.png)

**Comment by jgolbez** [comment]
I figured this out on my own referencing my old previous version build documentation. strings is part of the compiler so I installed g++ and zlib1g-dev and I now have all pre-reqs. Would be useful if we had a requirements.txt or some list of dependencies when running the configure script because the error is very unclear as to some dependencies, Others are very clear but some sort of package requirements file would alleviate it since it is also not documented.
